nuget.config

```diff
@@ -2,8 +2,9 @@
 
 <configuration>
   <packageSources>
-    <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="azure-appservice-staging" value="https://www.myget.org/F/azure-appservice-staging/api/v3/index.json" />
     <add key="azure-appservice" value="https://www.myget.org/F/azure-appservice/api/v3/index.json" />
```

Comment:
nit: should we remove these?

Reply:
I think we can do that in a separate PR. As part of the formal work to move our nuget packages. I only added the new feed so I could coordinate the public NuGet.org push together.